spacewalk/certs-tools/mgr_ssl_cert_setup.py

```diff
@@ -321,6 +322,10 @@ def getCertData(cert):
                 data["subjectKeyIdentifier"] = line.strip().upper()
             elif nextval == "authorityKeyIdentifier" and line.startswith("    keyid:"):
                 data["authorityKeyIdentifier"] = line[10:].strip().upper()
+            elif nextval == "authorityKeyIdentifier" and re.match(
+                r"^\s+[0-9A-Fa-f]{2}:.+$", line
```

Comment:
This is just checking if the value is directly the "hex" ID. 
openssl sometimes print this directly without identifier like "keyid:" as we match before.

Reply:
Well, whenever I create a regex, I put in an example of a string that the regex should match on, e.g. [0]. In that way, when the regex inevitably breaks because of some edge condition, it's easy to see what the old string was (without the edge case), and check the new regex against the old value as well.

[0] https://github.com/m-czernek/sos/blob/6e712a95b10210965c0c344cb72dd47225c448d5/sos/report/plugins/salt.py#L60-L63